File: mnegrete9.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int secretNumber, guess, i;

    secretNumber = 13;

    printf("Guess a number between 1 and 20:\n");

    scanf("%d", &guess);

    while (guess <= 1 || guess > 20) {
        printf("You input a number that is out of range (1-20) re-enter a valid number.\n");
        scanf("%d", &guess);
    }

    if (guess == secretNumber) {
        for (i = 0; i < guess; i++) {
            printf("You won and guessed the number correctly!\n");
        }
    }
    else {
        if (guess < secretNumber) {
            printf("You lost and your guess is lower than my number which was %d.\n", secretNumber);
        } else {
            printf("You lost and your guess is higher than my number which was %d.\n", secretNumber);
        }
    }

    return 0;
}

File: mnegrete9.java
//Moises Negrete
//lab #9



import java.util.Scanner;
import java.util.Random;

public class lab9 {
    public static void main(String[] args) {
        Scanner userinput = new Scanner(System.in);
        Random random = new Random();

        int randomNumber = random.nextInt(20) + 1;
        // +1 because random only goes to number -1
        int numberGuessed;

        System.out.println("guess a number between 1 and 20: \n");
        numberGuessed = userinput.nextInt();

        while (numberGuessed < 1 || numberGuessed > 20) {

            System.out.println("error number is out of reach please enter number in range\n");
            numberGuessed = userinput.nextInt();

        }

        if (numberGuessed == randomNumber) {
            for (int i = 0; i < numberGuessed; i++) {
                System.out.println("you won you guessed correctly");

            }
        } 
	else if (numberGuessed < randomNumber) {
            System.out.println("your guess is less than my number: my number was " + randomNumber);
        }
       	else	{
            System.out.println("you lost the guess is higher than my number: my number was" + randomNumber);
        }
    }
  }
